Lesson_23/array.js

```diff
@@ -0,0 +1,72 @@
+console.log(`____________ create Array _______________`);
+
+let Array = [371, null, 435, 5, false, 81, 101, 'name', 13, 40];
```

Comment:
wow! that's very huge mistake!
did I show the other way of creating arrays by l`et arr = new Array(1, 2, 3); `

what you are doing here it's just override the GLOBAL variable Array, so it can break the whole website:(

Reply:
how to avoid it:

1. don't make name of your own variables from capital letter
2. don't use the names of global key words which you saw already are going from JavaScript itself

Reply:
ок